app/dashboards/snap_application_dashboard.rb

```diff
@@ -0,0 +1,195 @@
+require "administrate/base_dashboard"
+
+class SnapApplicationDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
```

Comment:
same comment as above -- do we need these? on the one hand it's a nice shortcut for looking at docs, but it also makes the code harder to read because the file is longer. Your call!